opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -123,16 +125,18 @@ func resourceOpennebulaVirtualMachine() *schema.Resource {
 				Computed:    true,
 				Description: "Current LCM state of the VM",
 			},
-			"cpu":      cpuSchema(),
-			"vcpu":     vcpuSchema(),
-			"memory":   memorySchema(),
-			"context":  contextSchema(),
-			"disk":     diskVMSchema(),
-			"graphics": graphicsSchema(),
-			"nic":      nicVMSchema(),
-			"os":       osSchema(),
-			"vmgroup":  vmGroupSchema(),
-			"tags":     tagsSchema(),
+			"cpu":           cpuSchema(),
+			"vcpu":          vcpuSchema(),
+			"memory":        memorySchema(),
+			"context":       contextSchema(),
+			"disk":          diskVMSchema(),
+			"template_disk": diskComputedVMSchema(),
```

Comment:
maybe name attribute `computed_disk` instead of `template_disk` or rename the function to reflect the attribute name